adapters/silvermob/silvermob.go

```diff
@@ -18,6 +18,18 @@ type SilverMobAdapter struct {
 	endpoint *template.Template
 }
 
+func isValidHost(host string) bool {
+	validHosts := map[string]bool{
+		"eu":   true,
+		"us":   true,
+		"apac": true,
+	}
+
+	_, ok := validHosts[host]
+
+	return ok
```

Comment:
I think you might not wanna use a map here just to compare host value against 3 keys. This could be re-written as - 
`return host == "eu" || host == "us" || host == "apac"` instead of creating a map and then do lookup.
From performance perspective also, It would be better - I think.

Reply:
You're right, thanks